consumers.go

```diff
@@ -0,0 +1,58 @@
+package rabbithole
+
+import (
+	"net/url"
+)
+
+type BriefQueueInfo struct {
+	Name  string `json:"name"`
+	Vhost string `json:"vhost"`
+}
+
+type BriefChannelDetail struct {
+	ConnectionName string `json:"connection_name"`
+	Name           string `json:"name"`
+	Node           string `json:"node"`
+	Number         int    `json:"number"`
+	PeerHost       string `json:"peer_host"`
+	PeerPort       int    `json:"peer_port"`
+	User           string `json:"user"`
+}
+
+type ConsumerInfo struct {
+	Arguments      map[string]interface{} `json:"arguments"`
+	AckRequired    bool                   `json:"ack_required"`
```

Comment:
Ideally we should introduce an `AcknowledgementMode` enum type which has two values, `Manual` (acks are expected) and `Automatic` ([fire-and-forget](https://www.rabbitmq.com/confirms.html)). This would be clearer than a boolean with  this name. We could also introduce additional functions on `ConsumerInfo`. WDYT?

Reply:
I can try to implement the new type I don't have anything against it, I only did it that way so that the new types match what's returned by the API.
Regarding new functions on `ConsumerInfo` what new functions would you like to see ?

Reply:
I added `AcknowledgementMode` in `641510a` let me know if it is what you were looking for.